crates/driver/src/tests/setup/solver.rs

```diff
@@ -62,7 +62,7 @@ impl Solver {
                 order::Side::Buy if config.quote => {
                     "22300745198530623141535718272648361505980416".to_owned()
                 }
-                order::Side::Buy => match quote.order.fee_policy {
+                order::Side::Buy => match quote.order.fee_policy.first().unwrap() {
```

Comment:
Actually we could do this as a follow up, since right now we do not necessary need tests which checks the multiple fee calculation on the solver side :thinking: what do you guys think?

Reply:
If not too much work I'd do it now (or at least assert `quote.order.fee_policy.len() == 1` so that once a future developer crosses that bridge they fail explicitly in the place that needs a change.